alaoweb/2020.alaoweb.org
# Adds Preconference Speaker Info and Description

To test:
- Pull down the changes and build locally
- Visit `/preconference/` page
- Make sure the preconference info displays correctly

Closes #6
